railties/test/generators/plugin_generator_test.rb

```diff
@@ -148,6 +148,32 @@ def test_generating_adds_dummy_app_without_sprockets
     end
   end
 
+  def test_generator_for_yarn
```

Comment:
Rather than adding a test to `PluginGeneratorTest`, how about moving the test at `AppGeneratorTest` to `SharedGeneratorTests`?